ansible/roles/create-all-vms/tasks/by-family.yml

```diff
@@ -20,7 +21,7 @@
     apply:
       environment: "{{ ibm_env | default({}) }}"
   vars:
-    vm_name: "{{ gcp_instance_prefix }}-{{ normalized_collection_method }}-{{ item.1 }}-{{ job_id }}"
+    vm_name: "{{ gcp_instance_prefix }}-{{ item.1 | truncate(16, True, '') }}-{{ hashed_job_id }}"
```

Comment:
Should we just get rid of this `set_fact` block? 
https://github.com/stackrox/collector/blob/39f5c66645cd61417baa97067e06b9be24477499/ansible/roles/create-all-vms/tasks/main.yml#L3-L4

Reply:
Also, I'm not 100% sure we need to truncate the family, if I'm not mistaken, the longest name is used by fedora arm at 26 characters: `fedora-coreos-stable-arm64`.
Changing the prefix to `dev|ci` means we need 3 for that + the `-` + 26 for the family + another `-` + another 16 characters for the hash = 47 chars out of the 64 that is the limit. Just saying, we are probably good to leave the family as is.